src/components/calcite-shell-panel/calcite-shell-panel.tsx

```diff
@@ -115,6 +107,6 @@ export class CalciteShellPanel {
       mainNodes.reverse();
     }
 
-    return <Fragment>{mainNodes}</Fragment>;
+    return <div class={{ [CSS.container]: true, [CSS_UTILITY.rtl]: rtl }}>{mainNodes}</div>;
```

Comment:
@eriklharper re: our conversation

Reply:
@asangma this class can be removed now. `CSS_UTILITY`